FeedStoreChallenge/Infrastructure/Core Data/ManagedCache.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Ming-Ta Yang on 2021/6/12.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+internal final class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var feed: NSOrderedSet
+
+	internal var localFeed: [LocalFeedImage] {
+		return self.feed.compactMap { ($0 as? ManagedFeedImage)?.local }
+	}
```

Comment:
The convention in this codebase is to omit the default access-level of `internal`, so we should omit them.
```suggestion
final class ManagedCache: NSManagedObject {
	@NSManaged var timestamp: Date
	@NSManaged var feed: NSOrderedSet

	var localFeed: [LocalFeedImage] {
		return self.feed.compactMap { ($0 as? ManagedFeedImage)?.local }
	}
```